conda.md

```diff
@@ -0,0 +1,101 @@
+Conda
+====
+
+What is it?
+-----------
+
+[Conda](https://docs.conda.io/en/latest/) is a package/dependency management system that allows users to create reproducible computing environments for their analysis. It was originally developed for Python libraries, but has since been extended to also handle other software. It works with `environments` in which software is installed and that are (mostly) separated from the operating system. 
+
+Why should I use it?
+--------------------
+
+Conda allows you to separate the execution environment for different projects and explicitly control the version of each software in the environment independently of the operating system software management. This means you can have 
+- different versions of the same software for each project (it its own environment) without causing conflicts  
+- explicitly control the version of each software in your analysis to ensure reproducibility and prevent changes in results caused by version changes  
+- install software that might not be available from the package management of your operating system  
+
+Installation
+------------
+
+Conda can be installed as part of the [Miniconda](https://docs.conda.io/en/latest/miniconda.html) package. It works on Windows, MacOSX and Linux. See the website for further installation instructions.  
+
+Channels
+--------
+Software packages for conda (also called `recipes`) are provided in repositories called `channels`. Channels can be created by anyone and many organizations or even individual researchers maintain their own channels. By default conda will install packages from the main channel at [anaconda.com](https://repo.anaconda.com/pkgs/), but you specify other channels, when installing packages (see below). Useful channels include [`conda-forge`](https://anaconda.org/conda-forge) for community maintained packages, [`r`](https://anaconda.org/r) for R packages, and [`bioconda`](https://anaconda.org/bioconda) for bioinformatics packages. You can use the search on the [Anaconda main page](https://anaconda.org/) to search for channels, that provide a certain package. Just remember, that anyone can provide a channel and make sure that the channel is reliable.
```

Comment:
Unfortunately the whole paragraph is one line, a bit harder to read the diff. Anyway, there are a couple of superfluous commas here:
> ... but you [can] specify other channels when installing packages ...

> ... to search for channels that provide a certain package ...

> Just remember than anyone can provide ...

For the last sentence I'd add that you should see if they are trustworthy, and not trying to run a bitcoin miner on your machine :)

Reply:
I think some of the commas are not strictly necessary, but improve readability. So unless you think they are strictly wrong, I would like to keep them.